### PR TITLE
Correct few typos found.

### PR DESCRIPTION
**Pull request by kwilczynski**
This commit fixes few small typos found while reading the project code.

Signed-off-by: Krzysztof Wilczynski <kw@linux.com>

### CODE REVIEW

**Comment by stack72** [comment]
Thanks so much for this @kwilczynski :)

File: storage/directory.go [hunk 1]
```diff
@@ -102,7 +102,7 @@ type PutDirectoryInput struct {
 	DirectoryName string
 }
 
-// Put puts a directoy into the Triton Object Storage service is an idempotent
+// Put puts a director into the Triton Object Storage service is an idempotent
```
Comment (cheapRoc) [inline storage/directory.go]: This was still spelled wrong, forgot the `y`. 😄

**Comment by cheapRoc** [comment]
👍

File: storage/directory.go (hunk 1 above)
Comment (kwilczynski) [reply storage/directory.go]: Oh! That's my bad, sincere apologies!

I will get it fixed alongside different Pull Request.

**Pull request merged**